tests/fast-integration/README.md

```diff
@@ -46,6 +46,25 @@ npm install
 npm test
 ```
 
+
+#### To run eventing specific tests locally, follow these steps:
+1. Install dependencies:
+```bash
+cd kyma/tests/fast-integration
+npm install
+```
+
+2. Define SAP EventMesh Credentials JSON file path in `EVENTMESH_SECRET_FILE` environment variable.
```

Comment:
```suggestion
2. Define the Event Mesh Credentials JSON file path in the `EVENTMESH_SECRET_FILE` environment variable:
```
Since it's documentation on the open source GitHub, maybe we can omit SAP?